sonarr/assets/service_checks.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "agent_version": "7.40.1",
+        "integration": "Sonarr",
+        "check": "sonarr.can_connect",
+        "statuses": ["ok", "critical"],
+        "groups": [],
+        "name": "Can Connect",
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to the monitored sonarr instance. Returns `OK` otherwise."
```

Comment:
```suggestion
        "description": "Returns `CRITICAL` if the Agent is unable to connect to the monitored Sonarr instance. Returns `OK` otherwise."
```